itest/lib/log.ts

```diff
@@ -13,8 +13,6 @@ export const LOG = createLogger({
     })
   ),
   transports: [
-    new transports.Console({level: "info"}), // No idea how to make this runtime-configurable via command line. This has
```

Comment:
Are you removing this because "info" is already the default log level?

Reply:
This was just making the output on CI annoyingly long. The detailed info is in the file still, but doesn't clutter the console.